packages/generator/src/renderer.ts

```diff
@@ -166,9 +182,9 @@ function createHelpers(config: any): string {
 
 /**
  * Render the API Clients markdown file using the Handlebars template
- * @param {Map<string, WebApiBaseUnit[]>} apiFamilyMap
- * @param {Object.<string, RestApi[]} apiFamilyConfig
- * @returns {string} The rendered template
+ * @param apiFamilyMap -
+ * @param apiFamilyConfig -
```

Comment:
Missing description

Reply:
Yea, no one responded to me when I asked about this one in our chat :) I'm happy to add one, I just don't know what it is.

Reply:
What do you think the description should be?

Reply:
That's why you git blame and @ping. 😛 I was going to reply, but forgot!

```
* @param apiFamilyMap - Collection of API names and the AMF models associated with each API
* @param apiFamilyConfig - The API family config
```

Reply:
done